### ISSUE TITLE
Not working. 

### ISSUE THREAD

**Issue by brandbrilliance** [trakt/Plex-Trakt-Scrobbler#631]
I’ve installed it. Shows up under plugins. Authorised with the PIN from Trakt and saved the settings. It’s not scrobbling anything when looking in Trakt. It’s been weeks with no watch updates at all. Maybe put like some errors somewhere? Or what I can do to test it it working?

**Comment by karadoulis** [comment]
I have the same issue. Probably it stopped working...

**Comment by brandbrilliance** [comment]
Yeah, I even tried #622 and it didn't work either. Guess we stuck, as the other scrobblers are much more difficult to setup, and i haven't had success. I guess they just want you to pay for Plex and Trakt. Even if there is a manual script I can crontab myself every now and then would be helpful.

**Comment by karadoulis** [comment]
I tried https://github.com/Cornishman/Plex-Trakt-Scrobbler but also without success. I did tried tracksyncplex or whatever the name was but i didn't manage to make it work. Though it isn't nearly the same with this since you have to run it every time.

**Comment by karadoulis** [comment]
Hey @brandbrilliance, found a solution. I uninstalled the latest version for Plex Media Server and installed Version 1.27.2.5929. Plug in working fine with that version so i am guessing they messed up something in the new version of the server and it doesn't sync. Version 1.27.2.5929 works fine for me.

**Comment by brandbrilliance** [comment]
Great. But there doesn’t seem to be a list of older Versions to install? It only shows the latest version for macOS?

**Comment by brandbrilliance** [comment]
Ok I figured out how to construct the URL and managed to download it. Started it up but it refused to finish the server starting just hangs there now.

**Comment by karadoulis** [comment]
Did you clean the previous version? I did it with an uninstaller to delete every entry the previous version had. Might be a conflict between files?

**Comment by brandbrilliance** [comment]
Eventually started up. Scrobbling still not working. It’s so stupid. Thanks for trying.

**Comment by the-jeffski** [comment]
Can you copy the contents of the log file onto pastebin and add a link - might get a clue as to whats failing/working. Its been running fine for me for months and I did a clean install a couple of weeks back which worked OK.

**Comment by binamov** [comment]
My TV shows stopped scrobbling like two weeks ago. Movies still work.
Just did a fresh install and nothing works :(

**Comment by wizzardking** [comment]
Same problem here. But the following version does work properly on my end: https://github.com/linuxserver-labs/docker-plextraktsync

**Comment by tchirou** [comment]
Hi
Plex media server 1.30.1.6497 here
Just realised it stopped scrobbling tv shows 3 weeks ago. Movies still work.
Has someone found a fix ?
Thanks

**Comment by the-jeffski** [comment]
> Hi Plex media server 1.30.1.6497 here Just realised it stopped scrobbling tv shows 3 weeks ago. Movies still work. Has someone found a fix ? Thanks

What appears in the logs as an error? Mine has been fine - even recovered after the trakt outage.

**Comment by tchirou** [comment]
Nevermind, I installed this fork and it works fine now : https://github.com/Cornishman/Plex-Trakt-Scrobbler/tree/scrobble_fix_with_PlexOnlineToken_fix

**Comment by gitthangbaby** [comment]
Working in 2024 like every year before. Scrobbling and syncing. The only issue is it is constantly busy with backups and sync despite no events happen. I've had to disable "Sync - Run after library updates" to signifficantly decrease power consumption of the server. This plugin was more demanding than tons of other stuff running.

**Comment by karadoulis** [comment]
> Working in 2024 like every year before. Scrobbling and syncing. The only issue is it is constantly busy with backups and sync despite no events happen. I've had to disable "Sync - Run after library updates" to signifficantly decrease power consumption of the server. This plugin was more demanding than tons of other stuff running.

Which version did you use? I am still trying to make it work. I have tried it in 5 different servers but with no success. I unzip and copy the files to the folder. Then take a token code from traktv and copy it to the plugin. But it doesn't work. Which version are you using would be useful because I have tried every single one is in GitHub. As it seems the real problem is that plex doens't make a connection with traktv even after getting a token so there is no communication. And there are many people with this problem.

**Comment by gitthangbaby** [comment]
> And there are many people with this problem.

i'm still using v1.3.3 from [WebTools](https://github.com/ukdtom/WebTools.bundle). I've taken few files from the [forked repo](https://github.com/Cornishman/Plex-Trakt-Scrobbler) by hand - by checking what's updated "recently" (3 years ago). Plugin resides in /Plex Media Server/Plug-ins/Trakttv.bundle, data is in Plex Media Server/Plug-in Support/Data/com.plexapp.plugins.trakttv, /Plex Media Server/Plug-in Support/Databases etc. It all works just by copying files there. I changed server many times, and it all continues to work. In fact this is the most stable plugin connected to Trakt without constant reauthorization. I performed it now just to prove it works: copied the "Authentication PIN" from the link provided in the beginning of the Trakt.tv Settings. It's scroblling instantly during the playback.
My settings:

> [X] Scrobble
> Scrobble - Ignore duplicates for: 6 hours
> Sync - Watched: Synchronize (pull+push)
> Sync - Ratings: Synchronize (pull+push)
> Sync - Ratings - Conflict resolution: trakt
> Sync - Playback Progress: Synchronize (pull+push)
> Sync - Collection: Synchronize (pull+push)
> [ ] Sync - Clean collection
> Sync - Liked Lists (Beta): Synchronize (pull+push)
> Sync - Personal Lists (Beta) Sync: Synchronize (pull+push)
> Sync - Watchlist (Beta): Synchronize (pull+push)
> [X] Sync - Instant Actions <--- not implemented
> [ ] Sync - Run after library updates <--- causing high power consumption
> Sync - Run every: 12hours
> [ ] Sync - Defer until server is idle
> Sync - Idle delay: 15min
> ..
> Activity mode: Automatic
> Matcher: Plex Extended
> 

If something is not synced, [PlextTraktSync](https://github.com/Taxel/PlexTraktSync) can do the rest. It can scrobble too, but I find this plugin easier to set up.
To see what's happening watch the log: /PlexMediaServer/AppData/Plex Media Server/Logs/PMS Plugin Logs/com.plexapp.plugins.trakttv.log
Besides "Unable to read log file", "Ommitted attribute", i've no errors there.

**Comment by karadoulis** [comment]
I am using https://github.com/Cornishman/Plex-Trakt-Scrobbler and placed it on /Plex Media Server/Plug-ins/Trakttv.bundle. Besides that i did your settings for the plugin. I did the connection with traktv. Why are you using webtools since it is No longer being developed? Does it help? Do I have to copy anything else from the zip to other folder? Mine still not working :/ Thanks for the help. -tried your settings but it doesnt work :/

**Comment by gitthangbaby** [comment]
> I am using https://github.com/Cornishman/Plex-Trakt-Scrobbler and placed it on /Plex Media Server/Plug-ins/Trakttv.bundle. Besides that i did your settings for the plugin. I did the connection with traktv. Why are you using webtools since it is No longer being developed? Does it help? Do I have to copy anything else from the zip to other folder? Mine still not working :/ Thanks for the help. -tried your settings but it doesnt work :/

that's correct. Watch the log then. I'm using it because it works and is configurable right in Plex. The announcements of cease of plugin support mean nothing to me until the last plugin stops working.

**Comment by karadoulis** [comment]
> > I am using https://github.com/Cornishman/Plex-Trakt-Scrobbler and placed it on /Plex Media Server/Plug-ins/Trakttv.bundle. Besides that i did your settings for the plugin. I did the connection with traktv. Why are you using webtools since it is No longer being developed? Does it help? Do I have to copy anything else from the zip to other folder? Mine still not working :/ Thanks for the help. -tried your settings but it doesnt work :/
> 
> that's correct. Watch the log then. I'm using it because it works and is configurable right in Plex. The announcements of cease of plugin support mean nothing to me until the last plugin stops working.

I checked the trakt.log but i don't even know what I am looking for :P
